frontend/src/utils/backgroundFilter/applyBackgroundFilter/applyBackgroundFilter.ts

```diff
@@ -0,0 +1,73 @@
+import { hasMediaProcessorSupport, Publisher, VideoFilter } from '@vonage/client-sdk-video';
+import { UserType } from '../../../Context/user';
+import { BACKGROUNDS_PATH } from '../../constants';
+import { setStorageItem, STORAGE_KEYS } from '../../storage';
+
+export type ApplyBackgroundFilterParams = {
+  publisher?: Publisher | null;
+  backgroundSelected: string;
+  setUser?: (fn: (prev: UserType) => UserType) => void;
+  setBackgroundFilter?: (filter: VideoFilter | undefined) => void;
+  storeItem?: boolean;
+};
+
+/**
+ * Applies a background filter to the publisher.
+ * @param {ApplyBackgroundFilterParams} props - The props for the component.
+ *    @property {Publisher} publisher - The Vonage Publisher instance.
+ *    @property {string} backgroundSelected - The selected background option.
+ *    @property {Function} setUser - Optional function to update user state.
+ *    @property {Function} setBackgroundFilter - Optional function to set background filter state.
+ *    @property {boolean} storeItem - Optional flag to determine if the filter should be stored.
+ * @returns {Promise<void>} - A promise that resolves when the filter is applied or cleared.
+ * @throws {Error} - Throws an error if the publisher is not provided or if the backgroundSelected is invalid.
+ */
+export default async function applyBackgroundFilter({
```

Comment:
Nit, can we export as a `const` declaration instead? There might be other instances of this, too. Could you check?